src/pages/Liquidated.tsx

```diff
@@ -29,50 +22,31 @@ export function Liquidated() {
   };
 
   //  Queries for graph
-  const { data: tokenNamesData } = useSWR<AxiosResponse, AxiosError>(LIQUIDATION_GRAPH_TOKENS_QUERY, subQueryFetcher);
-  const tokenNamesResponse: TokenNamesResponse = tokenNamesData?.data.data;
-  const tokenNames = tokenNamesResponse?.vaultManagerMetrics.nodes.map((node) => node.liquidatingCollateralBrand) || [];
-
-  const { data: dailyMetricsData, isLoading: graphDataIsLoading } = useSWR<AxiosResponse, AxiosError>(
-    LIQUIDATION_DAILY_METRICS_QUERY(tokenNames),
-    subQueryFetcher,
+  const { data: vaultStateDailyData, isLoading: graphDataIsLoading } = useSWR<AxiosResponse, AxiosError>(
```

Comment:
I think we should also handle the error object returned by `useSwr.

We can handle it like:

```js
  if (error) {
    return <ErrorAlert value={error} />;
  }
```

Reply:
You can also do that for the `error` object returned by `useSwr` on line 15.

Reply:
will do

Reply:
btw im thinking a good idea would be to create a wrapper on all our component that catches any error and displays an error componenet instead. this would solve most of our issues with UI breaking and displaying an awkward log message

Reply:
> I've done a similar thing on the backend side 

While working on this, I noticed that I had to add numerous conditionals in my decorator to handle errors differently for various functions. Do you think it would be a good idea to create a wrapper around all the components?